src/Razor/src/rzls/rzls.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
This means OmniSharp has to install a .NET 7 SDK, right? or do we ship with our own?

Essentially, I'm sure this is safe because you know more about it than me, but it looks scary.